pkg/controller/direct/alloydb/cluster_mappings.go

```diff
@@ -0,0 +1,300 @@
+}
+func ContinuousBackupInfoObservedState_FromProto(mapCtx *direct.MapContext, in *pb.ContinuousBackupInfo) *krm.ContinuousBackupInfoObservedState {
+	if in == nil {
+		return nil
+	}
+	out := &krm.ContinuousBackupInfoObservedState{}
+	out.EncryptionInfo = EncryptionInfoObservedStateArray_FromProto(mapCtx, in.GetEncryptionInfo())
+	out.EnabledTime = direct.StringTimestamp_FromProto(mapCtx, in.GetEnabledTime())
+	out.Schedule = direct.EnumSlice_FromProto(mapCtx, in.Schedule)
+	out.EarliestRestorableTime = direct.StringTimestamp_FromProto(mapCtx, in.GetEarliestRestorableTime())
+	return out
+}
+func ContinuousBackupInfoObservedState_ToProto(mapCtx *direct.MapContext, in *krm.ContinuousBackupInfoObservedState) *pb.ContinuousBackupInfo {
+	if in == nil {
+		return nil
+	}
+	out := &pb.ContinuousBackupInfo{}
+	if len(in.EncryptionInfo) == 1 {
+		out.EncryptionInfo = EncryptionInfoObservedState_ToProto(mapCtx, in.EncryptionInfo[0])
+	}
+	out.EnabledTime = direct.StringTimestamp_ToProto(mapCtx, in.EnabledTime)
+	out.Schedule = direct.EnumSlice_ToProto[dayofweek.DayOfWeek](mapCtx, in.Schedule)
+	out.EarliestRestorableTime = direct.StringTimestamp_ToProto(mapCtx, in.EarliestRestorableTime)
+	return out
+}
+func TimeOfDay_FromProto(mapCtx *direct.MapContext, in *timeofday.TimeOfDay) *krm.TimeOfDay {
+	if in == nil {
+		return nil
+	}
+	out := &krm.TimeOfDay{}
+	out.Hours = direct.PtrTo(in.GetHours())
+	out.Minutes = direct.PtrTo(in.GetMinutes())
+	out.Seconds = direct.PtrTo(in.GetSeconds())
+	out.Nanos = direct.PtrTo(in.GetNanos())
+	return out
+}
+func TimeOfDay_ToProto(mapCtx *direct.MapContext, in *krm.TimeOfDay) *timeofday.TimeOfDay {
+	if in == nil {
+		return nil
+	}
+	out := &timeofday.TimeOfDay{}
+	out.Hours = direct.ValueOf(in.Hours)
+	out.Minutes = direct.ValueOf(in.Minutes)
+	out.Seconds = direct.ValueOf(in.Seconds)
+	out.Nanos = direct.ValueOf(in.Nanos)
+	return out
+}
+func MaintenanceUpdatePolicy_MaintenanceWindow_FromProto(mapCtx *direct.MapContext, in *pb.MaintenanceUpdatePolicy_MaintenanceWindow) *krm.MaintenanceUpdatePolicy_MaintenanceWindow {
+	if in == nil {
+		return nil
+	}
+	out := &krm.MaintenanceUpdatePolicy_MaintenanceWindow{}
+	out.Day = direct.Enum_FromProto(mapCtx, in.GetDay())
+	out.StartTime = TimeOfDay_FromProto(mapCtx, in.GetStartTime())
+	return out
+}
+func MaintenanceUpdatePolicy_MaintenanceWindow_ToProto(mapCtx *direct.MapContext, in *krm.MaintenanceUpdatePolicy_MaintenanceWindow) *pb.MaintenanceUpdatePolicy_MaintenanceWindow {
+	if in == nil {
+		return nil
+	}
+	out := &pb.MaintenanceUpdatePolicy_MaintenanceWindow{}
+	out.Day = direct.Enum_ToProto[dayofweek.DayOfWeek](mapCtx, in.Day)
+	out.StartTime = TimeOfDay_ToProto(mapCtx, in.StartTime)
+	return out
+}
+
+func UserPassword_FromProto(mapCtx *direct.MapContext, in *pb.UserPassword) *krm.UserPassword {
+	if in == nil {
+		return nil
+	}
+	out := &krm.UserPassword{}
+	out.User = direct.LazyPtr(in.GetUser())
+	// Password is unreadable.
+	// MISSING: Password
```

Comment:
Does the GCP service return the actual password?

Reply:
No.